.github/CODEOWNERS

```diff
@@ -97,6 +98,3 @@ ISSUE_TRIAGE_PROCESS.rst @eladkal
 # Migrations
 /airflow/migrations/ @ephraimbuddy
 /providers/fab/src/airflow/providers/fab/migrations/ @ephraimbuddy
-
-# AIP-72 - Task SDK
-/task-sdk/ @ashb @kaxil @amoghrajesh
```

Comment:
We still want this!

Reply:
Ah! Dont know how it got removed. Some rebase error i guess. Let me fix it